package.json

```diff
@@ -1,62 +1,40 @@
 {
   "name": "@yr/classlist",
   "description": "Ponyfill to fix element.classList browser inconsistencies",
-  "version": "2.0.0",
-  "author": "Alexander Pope <alexander.pope@nrk.no>",
+  "version": "3.0.0",
+  "author": "Alexander Pope <alexander.pope@nrk.no>, Simen Sægrov <simen.sagrov@nrk.no",
   "dependencies": {},
   "devDependencies": {
-    "babel-plugin-syntax-trailing-function-commas": "6.22.0",
-    "babel-plugin-transform-async-generator-functions": "6.24.1",
-    "babel-plugin-transform-async-to-generator": "6.24.1",
-    "babel-plugin-transform-es2015-arrow-functions": "6.22.0",
-    "babel-plugin-transform-es2015-block-scoped-functions": "6.22.0",
-    "babel-plugin-transform-es2015-block-scoping": "6.26.0",
-    "babel-plugin-transform-es2015-classes": "6.24.1",
-    "babel-plugin-transform-es2015-computed-properties": "6.24.1",
-    "babel-plugin-transform-es2015-destructuring": "6.23.0",
-    "babel-plugin-transform-es2015-duplicate-keys": "6.24.1",
-    "babel-plugin-transform-es2015-for-of": "6.23.0",
-    "babel-plugin-transform-es2015-function-name": "6.24.1",
-    "babel-plugin-transform-es2015-literals": "6.22.0",
-    "babel-plugin-transform-es2015-object-super": "6.24.1",
-    "babel-plugin-transform-es2015-parameters": "6.24.1",
-    "babel-plugin-transform-es2015-shorthand-properties": "6.24.1",
-    "babel-plugin-transform-es2015-spread": "6.22.0",
-    "babel-plugin-transform-es2015-sticky-regex": "6.24.1",
-    "babel-plugin-transform-es2015-template-literals": "6.22.0",
-    "babel-plugin-transform-es2015-unicode-regex": "6.24.1",
-    "babel-plugin-transform-es5-property-mutators": "6.24.1",
-    "babel-plugin-transform-exponentiation-operator": "6.24.1",
-    "babel-plugin-transform-object-rest-spread": "6.26.0",
-    "buddy": "6.x.x",
-    "chai": "^4.1.1",
-    "karma": "^1.7.0",
+    "@types/chai": "^4.1.2",
+    "@types/karma": "^1.7.3",
+    "@types/mocha": "^2.2.48",
+    "chai": "^4.1.2",
+    "karma": "^2.0.0",
     "karma-chai": "^0.1.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-mocha": "^1.3.0",
-    "karma-mocha-reporter": "^2.2.4",
-    "mocha": "*"
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-typescript": "^3.0.12",
+    "mocha": "*",
+    "prettier": "^1.11.1",
+    "ts-node": "^5.0.1",
+    "typescript": "^2.7.2"
   },
-  "main": "index.js",
+  "main": "dist/index",
```

Comment:
Not `dist/index.js`?

Reply:
When node supports modules natively, it will resolve .mjs files before .js files:
https://github.com/nodejs/node-eps/blob/master/002-es-modules.md